plugins/node/plugin.zsh

```diff
@@ -1,18 +1,32 @@
 # Color definitions
-GEOMETRY_COLOR_NODE_NPM_VERSION=${GEOMETRY_COLOR_NODE_NPM_VERSION:-black}
+GEOMETRY_COLOR_PACKAGER_VERSION=${GEOMETRY_COLOR_PACKAGER_VERSION:-black}
 
 # Symbol definitions
-GEOMETRY_SYMBOL_NODE_NPM_VERSION=${GEOMETRY_SYMBOL_NODE_NPM_VERSION:-"⬡"}
-GEOMETRY_NODE_NPM_VERSION=$(prompt_geometry_colorize $GEOMETRY_COLOR_NODE_NPM_VERSION $GEOMETRY_SYMBOL_NODE_NPM_VERSION) 
+GEOMETRY_SYMBOL_PACKAGER_VERSION=${GEOMETRY_SYMBOL_PACKAGER_VERSION:-"⬡"}
+GEOMETRY_NODE_PACKAGER_VERSION=$(prompt_geometry_colorize $GEOMETRY_COLOR_PACKAGER_VERSION $GEOMETRY_SYMBOL_PACKAGER_VERSION) 
+
+# Package manager
+GEOMETRY_NODE_DEFAULT_PACKAGE_MANAGER="npm"
 
 geometry_prompt_node_setup() {}
 
 geometry_prompt_node_render() {
-    if (( $+commands[node] )); then
-        GEOMETRY_NODE_VERSION="$(node -v 2> /dev/null)"
-        GEOMETRY_NPM_VERSION="$(npm --version 2> /dev/null)" 
-        echo "$GEOMETRY_NODE_NPM_VERSION $GEOMETRY_NODE_VERSION ($GEOMETRY_NPM_VERSION)"
+    if [[ ! $+commands[node] ]]; then 
+        return
+    fi
+
+    if [[ ! -f "$PWD/package.json" ]]; then
+        return
     fi
+
+    if [[ $+commands[yarn] && -f "$PWD/yarn.lock" ]]; then
```

Comment:
Something I'm thinking now is that `GEOMETRY_NODE_DEFAULT_PACKAGE_MANAGER` won't be restored to `npm` after entering a `yarn` based project. So, we should add a `else` statement to `GEOMETRY_NODE_DEFAULT_PACKAGE_MANAGER="npm"`.